assets/components/svg/svg.jsx

```diff
@@ -5,313 +5,227 @@
+
+  return (
+    <svg className="svg-video" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 12.728" preserveAspectRatio="xMinYMid">
+      <path d="M0 1.818v9.091l1.818 1.819h10.682v-12.728h-10.682zM18.75.165l-5 5v2.5l5 5.001h1.25v-12.501z" />
+    </svg>
+  );
+
+}
+
+// Just the 'P' of the PayPal logo.
+function SvgPaypalPLogo() {
+
+  return (
+    <svg className="svg-paypal-p-logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 23" preserveAspectRatio="xMinYMid">
+      <path d="M.555 20.444c-.58-.166-.503-.638-.439-1.035.239-1.578.497-3.156.749-4.727.713-4.477 1.432-8.96 2.154-13.449C3.174.287 3.471 0 4.42 0c2.8 0 5.6 0 8.387.096.976.065 1.93.312 2.813.728a4.171 4.171 0 0 1 2.246 4.926 15.908 15.908 0 0 0-2.736-.581c-2.18-.083-4.368 0-6.548 0-.749 0-1.026.242-1.142.977-.331 2.032-.658 4.066-.98 6.102-.059.357-.072.715-.104 1.073-.052.236-.11.473-.149.709-.316 1.955-.645 3.904-.935 5.865-.065.434-.252.569-.677.562-1.368-.013-2.697-.013-4.04-.013z" className="paypal-p-logo-0" />
+      <path d="M.555 20.444c1.348 0 2.69-.038 4.039 0 .425 0 .612-.127.677-.562.297-1.955.645-3.91.935-5.865.04-.236.097-.473.149-.709a1.23 1.23 0 0 1 1.168-.53h2.516a8.516 8.516 0 0 0 4.89-1.527c1.82-1.348 2.58-3.278 2.955-5.412 1.632.901 2.097 2.377 1.897 4.077a10.972 10.972 0 0 1-.742 2.92c-.955 2.293-2.845 3.398-5.265 3.698-.606.077-1.22.084-1.832.103-.73 0-1 .223-1.123.932-.245 1.432-.464 2.875-.69 4.313-.142.907-.361 1.099-1.29 1.099h-3.31c-.542 0-.742-.25-.645-.754.097-.505.168-1.067.264-1.693H2.806c-.748 0-1.516-.038-2.251-.09z" className="paypal-p-logo-1" />
+      <path d="M17.877 5.859c-.36 2.133-1.135 4.057-2.954 5.411a8.516 8.516 0 0 1-4.89 1.508c-.84.032-1.678 0-2.517 0a1.23 1.23 0 0 0-1.168.505c.033-.358.046-.722.104-1.074.322-2.036.65-4.07.98-6.101.123-.735.394-.971 1.142-.978 2.18 0 4.368-.057 6.549 0 .93.126 1.848.333 2.742.62a.272.272 0 0 1 .012.109z" className="paypal-p-logo-2" />
+    </svg>
+  );
+
+}
+
+// A Visa logo.
+function SvgVisaLogo() {
+
+  return (
+    <svg className="svg-visa-logo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 20" preserveAspectRatio="xMinYMid">
+      <path d="M32.29,20H0V17l27.18,0,5.16,0a0,0,0,0,1,0,0Z" className="visa-logo-0" />
+      <path d="M32.33,3,6,3,.1,3A.07.07,0,0,1,0,2.93V0H32.37Z" className="visa-logo-1" />
+      <path d="M12.74,5.62l-3.27,8.7H7.33L5.72,7.37a1,1,0,0,0-.48-.76,7.87,7.87,0,0,0-2-.74l0-.25H6.73a1,1,0,0,1,.93.89l.85,5,2.1-5.92Zm8.37,5.86c0-2.3-2.85-2.42-2.83-3.45,0-.31.27-.64.86-.73a3.46,3.46,0,0,1,2,.39l.35-1.85a5,5,0,0,0-1.89-.38c-2,0-3.4,1.18-3.41,2.88a2.8,2.8,0,0,0,1.77,2.37c.79.43,1.05.7,1,1.08,0,.58-.63.84-1.21.85a3.88,3.88,0,0,1-2.07-.55L15.34,14a5.59,5.59,0,0,0,2.25.46C19.71,14.46,21.1,13.29,21.11,11.48Zm5.27,2.84h1.87l-1.63-8.7H24.89a.92.92,0,0,0-.86.64l-3,8.07h2.12l.42-1.3h2.59Zm-2.25-3.08L25.19,8l.61,3.27Zm-8.5-5.62L14,14.32h-2l1.67-8.7Z" className="visa-logo-2" />
+    </svg>
+  );
+
+}
 
 
 // ----- Exports ----- //
 
-export default Svg;
+export {
```

Comment:
According, to the documentation, webpack's`Tree Shaking` feature should take care of deleting the unused exports via the **unused module export detection**. Nevertheless, I ran `yarn`'s `build-prod` task and I found some svgs that (I believe) we don't use in the bundle landing page. For example: twitter, facebook, amex and mastercard. Is it possible to correct setup this feature before merging, to be sure we don't run into with gigantic Svgs in the future?

Reply:
Yeah planned to do this a while back, now seems like a good time.

Done in a separate PR (#288) though, as it's potentially a larger change to the codebase and could break stuff.